package.json

```diff
@@ -29,25 +35,13 @@
   "lint-staged": {
     "*.js": "eslint"
   },
-  "clean-publish": {
```

Comment:
Does `clean-pkg-json` remove files, not needed for production release? Or will we have to handle it in CI?

Reply:
Oh, I see, files filed .. closing this comment